scss/modules/_lists.scss

```diff
@@ -190,3 +207,26 @@
     margin-left: -$gutter-width * 3;
   }
 }
+
+/// Default list tick pattern styles
+/// @group Lists
+%list-ticks {
```

Comment:
It looks like this placeholder is only used once. Would it be better to just have this inline where is it used or are you planning to use it in the future?